src/components/job/JobView.jsx

```diff
@@ -29,10 +40,96 @@ export default function JobView() {
       return <p>loading</p>;
     }
     return data ? (
-      <div className="flex flex-col">
-        <p className="text-bold text-2xl">{data.title}</p>
-        {`I am at src/components/job/JovView. This is job ${jobId}. `}
-        {JSON.stringify(data)}
+      <div className="flex flex-col card">
+        <div className="card-body items-center">
+          <p className="card-title text-2xl">{data.title}</p>
+          <p className="text-bold text-1xl">{data.company.name}</p>
+          <p className="text-bold text-1xl">
+            Posted:{" "}
+            {`${new Date(data.datePost.seconds * 1000)}`.split(" ").slice(0, 4).join(" ")}
+          </p>
+          <div className="flex w-full flex-col lg:flex-row">
+            <div className="flex flex-row mt-5 w-full lg:w-1/2 ">
+              <CalendarMonthIcon style={{ fontSize: iconSize }} />
+              <div className="ml-3">
+                <p className="font-bold">Starting Date:</p>
+                Date Created:
+                {`${new Date(data.datePost.seconds * 1000)}`
+                  .split(" ")
+                  .slice(0, 4)
+                  .join(" ")}
+              </div>
+            </div>
+            <div className="flex flex-row mt-5 w-full lg:w-1/2">
+              <WorkIcon style={{ fontSize: iconSize }} />
+              <div className="ml-3">
+                <p className="font-bold">Job Type:</p>
+                Based of Benefits:
+                {data.hasMedicalBenefit ? "Full-Time" : "Part-Time"}
```

Comment:
here do something like

Undecided fields: TBA
and add a comment called fixme: change it when the database is done

or just remove it

Reply:
Labeled "Database TBD"